packages/nextjs/src/server/errors.ts

```diff
@@ -28,11 +28,15 @@ Alternatively, you can set your own ignoredRoutes. See https://clerk.com/docs/ne
 (This log only appears in development mode)
 `;
 
-export const getAuthAuthHeaderMissing = () =>
-  'You need to use "authMiddleware" (or the deprecated "withClerkMiddleware") in your Next.js middleware file. You also need to make sure that your middleware matcher is configured correctly and matches this route or page. See https://clerk.com/docs/quickstarts/get-started-with-nextjs';
-
-export const authAuthHeaderMissing = () =>
-  "Clerk: auth() was called but it looks like you aren't using `authMiddleware` in your middleware file. Please use `authMiddleware` and make sure your middleware matcher is configured correctly and it matches this route or page. See https://clerk.com/docs/quickstarts/get-started-with-nextjs";
+export const getAuthAuthHeaderMissing = () => authAuthHeaderMissing('getAuth');
+
+export const authAuthHeaderMissing = (helperName = 'auth') =>
+  `Clerk: ${helperName}() was called but it looks like you aren't using authMiddleware in your middleware file. Please ensure the following:
+- Use authMiddleware in your middleware file.
```

Comment:
```suggestion
- Use authMiddleware in your Next.js Middleware file.
```

Reply:
done - dropping file from the first sentence as this is how it's used in the NextJS docs: https://nextjs.org/docs/app/building-your-application/routing/middleware